packages/element-api/src/express/routes/sidetree-v2/index.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+
+const router = express.Router();
+
+router.get('/sync', async (req, res, next) => {
+  try {
+    const sidetree = req.app.get('sidetree-v2');
+    await sidetree.sync();
```

Comment:
This is the kind of call I would expect to be impractical at scale... this is essentially, get me every DID Document that has ever been created...